items/src/main/scala/weco/api/items/Main.scala

```diff
@@ -20,6 +22,12 @@ import weco.sierra.http.SierraSource
 import weco.sierra.typesafe.SierraOauthHttpClientBuilder
 
 import scala.concurrent.ExecutionContext
+//}
+class LondonClock {
+  def now(): ZonedDateTime =
+    ZonedDateTime.now.withZoneSameLocal(ZoneId.of("Europe/London"))
+  def getHour: Int = now().getHour
+}
```

Comment:
Couple of things about this class; firstly I would move it to its own file (probably in `services`) as you're also using it elsewhere.

When we define it as a `LondonClock` here, name it in a parameter as `clock`, and then write`clock.now()` and `clock.getHour` we kind of lose the context of the timezone: in `SierraItemUpdater` it's not clear what time the clock is giving you. The important thing is that this clock gives you the time at Wellcome Collection.

Maybe we could do something like this:
```scala
class VenueClock {
  def timeAtVenue: ZonedDateTime = ??? // London time
  def hourAtVenue: Int = ??? // as above
}
```

then when we use it, it's always clear where the time is coming from: `clock.timeAtVenue`

Reply:
Replaced by java Clock